yarn.lock

```diff
@@ -11019,6 +11019,11 @@ ws@^7.0.0, ws@^7.2.1:
   resolved "https://registry.yarnpkg.com/ws/-/ws-7.2.1.tgz#03ed52423cd744084b2cf42ed197c8b65a936b8e"
   integrity sha512-sucePNSafamSKoOqoNfBd8V0StlkzJKL2ZAhGQinCfNQ+oacw+Pk7lcdAElecBF2VkLNZRiIb5Oi1Q5lVUVt2A==
 
+ws@^7.2.1:
```

Comment:
This is a weird addition since nothing else changed. Did we update `ws` explicitly?

Reply:
I don't remember updating `ws` directly. Is it possible that it was included in another installation? `jest-jsdom-environment-sixteen` perhaps?

Reply:
Let's debug this: Let's try:

1. Removing this then run `yarn install` and see if there is a diff
2. Also, removing this, removing `node_modules`, and then run `yarn install` and see if there is a diff

Reply:
When asking yarn why ws:

```
➜  design-system git:(webinf-dates-you-can-pick-some-can-t-be-made) ✗ yarn why ws
yarn why v1.17.3
[1/4] 🤔  Why do we have the module "ws"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "ws@7.2.1"
info Has been hoisted to "ws"
info Reasons this module exists
   - Hoisted from "jest-environment-jsdom-sixteen#jsdom#ws"
   - Hoisted from "jest#@jest#core#jest-config#jest-environment-jsdom#jsdom#ws"
info Disk size without dependencies: "144KB"
info Disk size with unique dependencies: "144KB"
info Disk size with transitive dependencies: "144KB"
info Number of shared dependencies: 0
=> Found "webpack-dev-server#ws@6.2.1"
info This module exists because "react-styleguidist#webpack-dev-server" depends on it.
info Disk size without dependencies: "136KB"
info Disk size with unique dependencies: "240KB"
info Disk size with transitive dependencies: "240KB"
info Number of shared dependencies: 1
✨  Done in 0.67s.
```

Reply:
It looks like it's included with `jest-environment-jsdom-sixteen`